Billing manager docs (#1817)
pulumi/pulumi-hugo: themes/default/content/docs/intro/pulumi-service/billing-managers.md

---
title: "Billing Managers"
meta_desc: Overview of Billing Managers in the Pulumi Service.
menu:
  intro:
    parent: pulumi-service
    weight: 11
---

> The Billing Manager role provides customers the ability to have someone in your organization manage billing operations without granting them any additional permissions to view or modify your stacks, policies, or other organizational settings.

Reviewer: ```suggestion
The Billing Manager role gives customers the ability to have someone in their Pulumi Organization manage billing operations without granting them any additional permissions to view or modify your stacks, policies, or other organizational settings.
```

> Any organization admin can view, invite, and remove Billing Managers from their organization.

Reviewer: ```suggestion
Any Organization Admin can view, invite, and remove Billing Managers from their organization.
```

> Team Editions of the Pulumi Service are limited to 1 Billing Manager at a time. Enterprise and Business Critical subscriptions can have unlimited Billing Managers in their organization.

Reviewer: ```suggestion
Team Editions of the Pulumi Service are limited to 1 Billing Manager at a time. Enterprise and Business Critical subscriptions can have unlimited Billing Managers in their organization. Visit our [Pricing page](https://www.pulumi.com/pricing) to learn more about Pulumi Service Editions.
```

## Inviting a Billing Manager to an Organization

Billing Manager invitations work much the same way member invitations do, but they are managed on the Billing and Usage page.

To invite a new Billing Manager:

1. Select **Settings**.

> 1. Select **Billing & usage**.

Reviewer: ```suggestion
2. Select **Billing & usage**.
```

Reviewer 2: if we change this numbering, if we ever add future values we will have to adjust the numbering using all 1.s means this will be done automatically for us.

> 1. To invite a new Billing Manager using their email address, select **Invite Billing Managers**. After the new Billing Manager is invited, they will receive an email allowing them to accept the invite.

Reviewer: ```suggestion
3. To invite a new Billing Manager using their email address, select **Invite Billing Managers**. After the new Billing Manager is invited, they will receive an email allowing them to accept the invite.
```

> 1. To invite a new Billing Manager by sharing a link, select **Copy Billing Manager invite link**. This will copy a link to your clipboard that you can share directly.

Reviewer: ```suggestion
4. To invite a new Billing Manager by sharing a link, select **Copy Billing Manager invite link**. This will copy a link to your clipboard that you can share directly.
```

As with member invitation links, new Billing Manager invitation links never expire and may only be used one time.

Billing Managers are not included in an organization's member count. For example, if your organization has one admin, two members, and a Billing Manager, the total member count will be three, not four.

## Viewing Billing Managers

Billing Managers and pending Billing Manager invitations can be viewed by going to the organization's Settings page, then selecting Billing & usage from the menu. As with the Members page, this table displays all Billing Managers belonging to your organization, their email address, and when they accepted the invite. For pending Billing Manager invitations, the invitation status column will include the date an invitation was sent, and will note if a potential Billing Manager encountered an error while attempting to accept an invitation.

> Pending invites and Billing Manager emails and invite status are only viewable by organization admins. Billing Manager will be able to see other Billing Managers, but only their names.

Reviewer: ```suggestion
Pending invites and Billing Manager emails and invite status are only viewable by Organization Admins. Billing Manager will be able to see other Billing Managers, but only their names.
```
